nvflare/fuel/f3/drivers/aio_grpc_driver.py

```diff
@@ -374,38 +382,9 @@ def shutdown(self):
     def get_urls(scheme: str, resources: dict) -> (str, str):
         secure = resources.get(DriverParams.SECURE)
         if secure:
-            scheme = "grpcs"
+            if use_aio_grpc():
+                scheme = "grpcs"
+            else:
+                scheme = "agrpcs"
```

Comment:
Should it be "grpc" VS "nagrpc"?

Reply:
The default scheme of FLARE is always "grpc", and we don't want to change that.  On the other hand, each driver must provide its schemes supported. Since we want to have the flexibility of choosing which driver to use for the scheme "grpc", we need to be able to choose it dynamically based on config of "use_aio_grpc".

When this is set to True, then the AioGrpcDriver will take the scheme "grpc", and the  GrpcDriver will take the scheme of "nagrpc" (two drivers cannot have the same scheme). If use_aio_grpc is False, then the GrpcDriver will take the scheme "grpc", and the AioGrpcDriver will take the scheme "agrpc".

When "grpc" is taken by one driver, you could let the other driver always take the same name like "xgrpc". But it is more clear to have their meaning name (agrpc = aio-grpc, nagrpc = non-aio-grpc).

Reply:
Got it, thanks for the explanation!

It might be hard to get these points for regular users, but we could enhance that later if this gets more interests from them.